__tests__/web/keyboard/index.spec.ts

```diff
@@ -0,0 +1,495 @@
+        },
+        Object {
+          "code": "Digit2",
+          "key": "2",
+          "keyCode": 50,
+          "modifiers": Object {},
+          "timestamp": 1000,
+          "type": "keyup",
+        },
+        Object {
+          "code": "Digit3",
+          "key": "3",
+          "keyCode": 51,
+          "modifiers": Object {},
+          "timestamp": 1100,
+          "type": "keydown",
+        },
+        Object {
+          "code": "Digit3",
+          "key": "3",
+          "keyCode": 51,
+          "modifiers": Object {},
+          "timestamp": 1200,
+          "type": "keypress",
+        },
+        Object {
+          "data": "3",
+          "timestamp": 1300,
+          "type": "text_input",
+          "value": "",
+        },
+        Object {
+          "timestamp": 1400,
+          "type": "text_change",
+          "value": "123",
+        },
+        Object {
+          "code": "Digit3",
+          "key": "3",
+          "keyCode": 51,
+          "modifiers": Object {},
+          "timestamp": 1500,
+          "type": "keyup",
+        },
+        Object {
+          "code": "Digit4",
+          "key": "4",
+          "keyCode": 52,
+          "modifiers": Object {},
+          "timestamp": 1600,
+          "type": "keydown",
+        },
+        Object {
+          "code": "Digit4",
+          "key": "4",
+          "keyCode": 52,
+          "modifiers": Object {},
+          "timestamp": 1700,
+          "type": "keypress",
+        },
+        Object {
+          "data": "4",
+          "timestamp": 1800,
+          "type": "text_input",
+          "value": "",
+        },
+        Object {
+          "timestamp": 1900,
+          "type": "text_change",
+          "value": "1234",
+        },
+        Object {
+          "code": "Digit4",
+          "key": "4",
+          "keyCode": 52,
+          "modifiers": Object {},
+          "timestamp": 2000,
+          "type": "keyup",
+        },
+        Object {
+          "code": "Digit5",
+          "key": "5",
+          "keyCode": 53,
+          "modifiers": Object {},
+          "timestamp": 2100,
+          "type": "keydown",
+        },
+        Object {
+          "code": "Digit5",
+          "key": "5",
+          "keyCode": 53,
+          "modifiers": Object {},
+          "timestamp": 2200,
+          "type": "keypress",
+        },
+        Object {
+          "data": "5",
+          "timestamp": 2300,
+          "type": "text_input",
+          "value": "",
+        },
+        Object {
+          "timestamp": 2400,
+          "type": "text_change",
+          "value": "12345",
+        },
+        Object {
+          "code": "Digit5",
+          "key": "5",
+          "keyCode": 53,
+          "modifiers": Object {},
+          "timestamp": 2500,
+          "type": "keyup",
+        },
+      ]
+    `);
+    expect(curr.type).toMatchInlineSnapshot(`"TEXT"`);
+    expect(curr.target).toMatchInlineSnapshot(
+      {
+        id: expect.any(String),
```

Comment:
what is this for?

Reply:
id is random() and cannot predict, tell jest not to match this property when comparing snapshot.